src/messages/MessageElement.hpp

```diff
@@ -39,6 +40,9 @@ enum class MessageElementFlag {
     BttvEmoteImage = (1 << 6),
     BttvEmoteText = (1 << 7),
     BttvEmote = BttvEmoteImage | BttvEmoteText,
+
+    ChannelPointReward = (1 << 8),
```

Comment:
I've squeezed it into a free value since the underlying type for the enum is only 32 bits in some systems.
I've tried to change the underlying type to something like a `std::uint64_t` but I encountered some issues with the `FlagEnum` container which I didn't know how to solve.